frontend/local/src/containers/patients/detail/record.js

```diff
@@ -109,12 +112,17 @@ class HealthRecord extends React.Component {
                                                     <Collapse isOpen={this.state[meta.name] ? this.state[meta.name]["complementaryDiagnoses"] : false}>
                                                         <dl>
                                                             {data.diagnoses.map((diagnosis, i) => {
-                                                                return (i !== 0) && (
-                                                                    <React.Fragment key={"diagnosis" + i}>
-                                                                        <dt>{diagnosis.diagnosis ? (diagnosis.diagnosis.label || "Diagnosis") : "Diagnosis"}</dt>
-                                                                        <dd>{diagnosis.comment}</dd>
-                                                                    </React.Fragment>
-                                                            )})}
+                                                                return (
+                                                                    i !== 0 && (
+                                                                        <React.Fragment key={"diagnosis" + i}>
+                                                                            <dt>
+                                                                                {diagnosis.diagnosis ? diagnosis.diagnosis.label || "Diagnosis" : "Diagnosis"}
```

Comment:
could be more readable. Maybe you could replace it with something like `_.get(diagnosis, 'diagnosis.label', 'Diagnosis')`.